install.js

```diff
@@ -28,7 +28,7 @@ if (platform === 'linux') {
     platform += '32'
   }
 } else if (platform === 'darwin') {
-  platform = 'mac32'
+  platform = 'mac64'
```

Comment:
I know it's not super common for people to be running 32-bit OSX today, but should this also have a conditional check for `process.arch == x64`?